src/config/dynamic-theme-fixes.config

```diff
@@ -1356,6 +1356,25 @@ INVERT
 
 ================================
 
+babyem.co.uk
+
+INVERT
+section h1
```

Comment:
You can remove the `section *` things, they were a bug in the variables implementation and fixed in the next release. Relevant commit: https://github.com/darkreader/darkreader/commit/b74917d661fdbfb9a31710621552008a90a14a21

I'm not sure if the 2 non-section selectors are also affected by this.